### PR TITLE
Es default installation

### PR DESCRIPTION
**Pull request by elioschmutz**
This PR implements a full configured solr installation.

You can use the ``development.cfg`` to install solr. You don't have to configure something else to work with it.

bildschirmfoto 2015-12-17 um 09 30 58
bildschirmfoto 2015-12-17 um 09 31 29
bildschirmfoto 2015-12-17 um 09 32 04
bildschirmfoto 2015-12-17 um 09 32 09

In addition it provides some basic css for default plone and an dynamic scss integration for ftw.theming.

### CODE REVIEW

**Comment by maethu** [comment]
There are two livesearch.js files, are both required?

- https://github.com/4teamwork/ftw.solr/blob/es-default-installation/ftw/solr/browser/resources/livesearch.js
- https://github.com/4teamwork/ftw.solr/blob/es-default-installation/ftw/solr/skins/ftw_solr/livesearch.js

File: ftw/solr/browser/search.pt [hunk 1]
```diff
@@ -6,57 +6,45 @@
...
+
+    <div id="content-core" metal:fill-slot="main">
+
+      <h1 class="hiddenStructure"
+          i18n:translate="label_search">
+          Suche
+      </h1>
+
+      <div class="row">
+          <div class="searchLeftPart">
```
Comment (maethu) [inline ftw/solr/browser/search.pt]: Please omit this div (searchLeftPart), since there is no right part :-)

**Comment by maethu** [comment]
What about french and italien translations? you may get theme from bern.web.

File: ftw/solr/browser/search.py [hunk 2]
```diff
@@ -164,8 +162,3 @@ def breadcrumbs(self, item):
             empty = {'absolute_url': '', 'Title': unicode('…', 'utf-8')}
             breadcrumbs = [breadcrumbs[0], empty] + breadcrumbs[-maxb+1:]
         return breadcrumbs
-
-
-class SearchBoxViewlet(common.SearchBoxViewlet, FacetMixin):
```
Comment (maethu) [inline ftw/solr/browser/search.py]: @elioschmutz  There are several other packages, ex. plonetheme.onegov, which relay on this class for overrides.
We may simply add `from ftw.solr.viewlets.searchbox import SearchBoxViewlet` to this module for backwars compatibility? any other suggestion?

Comment (elioschmutz) [inline ftw/solr/browser/search.py]: :+1: I use a deprecatedfunction for the SearchBoxViewlet.

**Comment by elioschmutz** [comment]
@maethu 

- [x] Translate fr and it
- [x] Use deprecation function for backwards compatibility
- [x] remove unused livesearch.js

**Pull request merged**